### DISCUSSION TITLE
should we remove duplicates or just mark duplicates in the bam file for archR?

### DISCUSSION THREAD

**Discussion by YichaoOU** [GreenleafLab/ArchR#2056] [Questions / Documentation]
Hello,

I have marked the duplicates per cell using picard. Will archR take care of these duplicates when reading the bam? Or I should remove the PCR duplicates before asking archR to process?

Thanks,
Yichao

**Comment by rcorces** [comment]
You should familiarize yourself with the `bamFlag` parameter which is used for this and similar filtering purposes
